www/search.spt

```diff
@@ -154,12 +158,19 @@ if query:
                         len(repositories)) }}</h3>
         % for repo in repositories
             % set owner = repo.owner
+            % set blur = owner.is_unsettling and 'always_view_unsettling' not in request.headers.cookie
             <div class="paragraph">
-                <a href="/{{ owner.username }}/" class="avatar-inline">
-                    {{ avatar_img(owner, size=28) }}
-                </a>
+                <div class="repo-search-display {{ 'repo-search-blur' if blur and owner.id != user.id }}">
+                    <a href="/{{ owner.username }}/" class="avatar-inline">
+                        {{ avatar_img(owner, size=28) }}
+                    </a>
+                        {{ _(
+                        "{username} ",
+                        username=('<a href="/{0}/">{0}</a>'|safe).format(owner.username),
+                    ) }}
+                </div>
                 {{ _(
-                    "{username} has a repository named {repo_name} in their {platform} account",
+                    "has a repository named {repo_name} in their {platform} account",
                     username=('<a href="/{0}/">{0}</a>'|safe).format(owner.username),
```

Comment:
If the `username` variable isn't used in the string anymore, then it should be removed from the arguments passed to the translation function. However, the larger problem here is that this change assumes that the username is at the beginning of the sentence in every language. A cleaner solution would be to generate the possibly-blurred avatar+username box with a macro or [block assignment](https://jinja.palletsprojects.com/en/3.0.x/templates/#block-assignments), and pass it to the translation function in the `username` argument.